# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../netsniff-ng/0003-add-DESTDIR.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/Makefile	2014-04-07 19:02:33.730426413 +0200
+++ b/Makefile	2014-04-07 19:03:49.340188075 +0200
@@ -28,6 +28,9 @@
 # Disable if you don't want it
 CCACHE ?= $(CONFIG_CCACHE)
 
+# Location of installation destination directory
+DESTDIR=
+
 # Location of installation paths.
 SBINDIR = $(PREFIX)/usr/sbin
 INCDIR = $(PREFIX)/usr/include
--- a/Template	2014-04-07 18:59:00.501491038 +0200
+++ b/Template	2014-04-07 19:00:37.507754557 +0200
@@ -8,14 +8,14 @@
   $(1)_clean: $(1)_clean_custom
 	$(Q)$$(call RM,$(1)/*.o $(1)/$(1) $(1)/*.gz)
   $(1)_do_install:
-	$(Q)$$(call INSTX,$(1)/$(1),$$(SBINDIR))
+	$(Q)$$(call INSTX,$(1)/$(1),$$(DESTDIR)$$(SBINDIR))
 	$(Q)$(GZIP) $(1).8 > $(1)/$(1).8.gz
-	$(Q)$$(call INSTX,$(1)/$(1).8.gz,$$(MAN8DIR))
-	$(Q)$$(foreach file,$$($(1)-confs),$$(call INST,$$(file),$$(ETCDIRE));)
+	$(Q)$$(call INSTX,$(1)/$(1).8.gz,$$(DESTDIR)$$(MAN8DIR))
+	$(Q)$$(foreach file,$$($(1)-confs),$$(call INST,$$(file),$$(DESTDIR)$$(ETCDIRE));)
   $(1)_install: $(1)_do_install $(1)_post_install
   $(1)_uninstall: $(1)_uninstall_custom
-	$(Q)$$(call RM,$$(SBINDIR)/$(1))
-	$(Q)$$(call RM,$$(MAN8DIR)/$(1).8.gz)
+	$(Q)$$(call RM,$$(DESTDIR)$$(SBINDIR)/$(1))
+	$(Q)$$(call RM,$$(DESTDIR)$$(MAN8DIR)/$(1).8.gz)
   $(1)/%.yy.o: $(1)/%.yy.c
 	$$(CC) $$(CFLAGS) -o $$@ -c $$<
   $(1)/%.tab.o: $(1)/%.tab.c
--- a/trafgen/Makefile	2014-04-07 20:02:43.197995226 +0200
+++ b/trafgen/Makefile	2014-04-07 20:03:13.946689298 +0200
@@ -31,7 +31,7 @@
 trafgen-confs =	trafgen_stddef.h
 
 trafgen_post_install:
-	$(Q)mv $(ETCDIRE)/trafgen_stddef.h $(ETCDIRE)/stddef.h
+	$(Q)mv $(DESTDIR)$(ETCDIRE)/trafgen_stddef.h $(DESTDIR)$(ETCDIRE)/stddef.h
 
 trafgen_clean_custom:
 	$(Q)$(call RM,$(BUILD_DIR)/*.h $(BUILD_DIR)/*.c)
